x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/table/title_with_explanation.tsx

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { ReactElement } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { EuiPopover, EuiIcon, EuiText } from '@elastic/eui';
+import { useBoolean } from '@kbn/react-hooks';
+import { css } from '@emotion/react';
+
+interface Props {
+  name?: string;
+  description: ReactElement | string;
+  formula: string;
+  'data-test-subj'?: string;
+}
+
+function TitleWithPopoverExplanation(props: Props) {
+  const { name, description, formula } = props;
+  const [isPopoverOpen, { off: closePopover, toggle: togglePopover }] = useBoolean(false);
+
+  return (
+    <>
+      {name && <>{name}</>}
```

Comment:
nit: The component is `TitleWithPopoverExplanation` but we use `name`, I was looking for the variable `title` when I checked the code. But that could be just me 🐒